example/reactor/tiny_main.go

```diff
@@ -0,0 +1,26 @@
+//go:build !std
+// +build !std
+
+package main
+
+import (
+	"os"
+
+	"github.com/extism/go-pdk"
+	_ "github.com/extism/go-pdk/pdk_reactor"
```

Comment:
nit: can we rename this package to `wasi-reactor` in the path, and `reactor` within the package?

Reply:
good call